### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by germartinez**
Update README file with:
- Title
- Image
- Relevant information

### CODE REVIEW

**Review by iamacook — APPROVED** [review]
Nice to remove the template, but do we not need any further info here?

**Comment by germartinez** [comment]
@iamacook I just added more relevant information

File: README.md [hunk 1]
```diff
@@ -1,93 +1,51 @@
...
-- [x] One icon favicons
-- [x] Manifest support
-- [x] Sitemap support
-- [x] Offline support
-- [x] Easy to use media queries
-- [x] Using Typescript
-- [x] Using styled-components
-- [x] Using Prettier
-- [x] Using tslint
+- Production: https://gnosis-safe.io/
```
Comment (iamacook) [inline README.md:10]: I would remove the trailing slash for consistency with the other links

File: README.md [hunk 2]
```diff
@@ -1,93 +1,51 @@
...
+- Staging: http://safe-team.staging.gnosisdev.com
+- Development: http://safe-team.dev.gnosisdev.com
 
-## 🚀 Quick start
+## Getting Started
 
-1.  **Create a Gatsby site.**
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
```
Comment (iamacook) [inline README.md:16]: "These instructions will help you..."

File: README.md [hunk 3]
```diff
@@ -1,93 +1,51 @@
...
-[![Netlify Status](https://api.netlify.com/api/v1/badges/184bab96-1bf5-41f0-bb14-df00c5e9ec68/deploy-status)](https://app.netlify.com/sites/gatsby-starter-gnonce/deploys)
+```
+yarn start
+```
 
-Get your Gatsby site in 1 min.
+### Building
 
-Netlify is a unified platform that automates your code to create high-performant, easily maintainable sites and web apps. They provide continuous deployment (Git-triggered builds), an intelligent, global CDN, full DNS (including custom domains), automated HTTPS, asset acceleration, and a lot more.
+Te get a complete bundle use:
```
Comment (iamacook) [inline README.md:39]: "To"

File: README.md (hunk 2 above)
Comment (germartinez) [reply README.md:16]: "[...] help you a copy [...]"?

Comment (iamacook) [reply README.md:16]: "These instructions will help you get you a copy of the project up and running on your local machine for development and testing purposes."

**Pull request merged**